website/pages/docs/internals/telemetry.mdx

```diff
@@ -322,6 +322,75 @@ These metrics relate to the supported [storage backends][storage-backends].
 | `vault.zookeeper.delete`    | Duration of a DELETE operation against the [ZooKeeper storage backend][zookeeper-storage-backend]                      | ms   | summary |
 | `vault.zookeeper.list`      | Duration of a LIST operation against the [ZooKeeper storage backend][zookeeper-storage-backend]                        | ms   | summary |
 
+
+## Integrated Raft Storage Metrics
+
+These metrics relate to raft based [integrated storage][integrated-storage].
+
+| Metric                                          | Description                                                                                 | Unit | Type    |
+| :---------------------------------------------- | :------------------------------------------------------------------------------------------ | :--- | :------ |
+| `vault.raft.apply`															| Number of logs applied                                                                      | ms   | counter |
```

Comment:
ms is the wrong Unit for counter types. Can you update all the counters you've added in this PR? You can copy what consul uses for these, for instance for `raft.apply`: "raft transactions / interval"

Reply:
Could you also use the descriptions from consul's docs as well? They are more descriptive, for instance for this metric they have: 
```
This counts the number of Raft transactions occurring over the interval, which is a general indicator of the write load on the Consul servers.
```
Just be sure to s/Consul/Vault/

Reply:
done